app/src/main/java/fr/free/nrw/commons/upload/mediaDetails/UploadMediaDetailFragment.java

```diff
@@ -297,6 +299,32 @@ public void onImageProcessed(UploadItem uploadItem, Place place) {
         setDescriptionsInAdapter(descriptions);
     }
 
+    /**
+     * Shows popup if any nearby location needing pictures matches uploadable picture's GPS location
+     * @param uploadItem
+     * @param place
+     */
+    @SuppressLint("StringFormatInvalid")
+    @Override
+    public void onNearbyPlaceFound(UploadItem uploadItem, Place place) {
+        DialogUtil.showAlertDialog(getActivity(),
+                getString(R.string.upload_nearby_place_found_title),
+                String.format(Locale.getDefault(),
+                        getString(R.string.upload_nearby_place_found_description),
+                        place.getName()),
+                () -> {
```

Comment:
Is this intended to be empty?

Reply:
yup. there's no action on cancel. the dialog is dismissed automatically.